src/main/java/frc/trigon/robot/utilities/Conversions.java

```diff
@@ -0,0 +1,20 @@
+package frc.trigon.robot.utilities;
+
+public class Conversions {
+
+        /**
+        * Converts voltage to compensated power
+        *
+        * The voltage compensation saturation will determine what voltage represents 100% output.
+        *
+        * The compensated power is the power resulting from turning the voltage off and on without stopping.
+        * It's basically the voltage divided by the saturation of the compensation.
+        *
+        * @param voltage the voltage
+        * @param voltageCompensationSaturation the saturation of the compensation
+        * @return the compensated power
```

Comment:
nope. please write proper documentation

Reply:
What is better than this @lycaon7?

Reply:
you figure it out. but what you have currently written explains nothing

Reply:
I really don't know. Maybe you were right, I don't know what I'm doing :(